server/storage/mvcc/watcher.go

```diff
@@ -58,6 +58,13 @@ type WatchStream interface {
 	// of the watchers since the watcher is currently synced.
 	RequestProgress(id WatchID)
 
+	// RequestProgressAll requests a progress notification for the
+	// entire watcher group.  The responses will be sent through
```

Comment:
Please change the comment to something like below, (not a native English speaker, please feel free to adjust the wording.)
```
RequestProgressAll requests a progress notification for the
all watchers (substreams) which share the same gRPC stream. 
......
```

Reply:
Re-worded it. Felt like it might be the wrong level of abstraction to talk about gRPC (strictly speaking we don't know where the WatchResponses go) and substreams (that's a client-side idea?), so I left it just at "share the same stream".